src/content-tags-drawer/data/apiHooks.jsx

```diff
@@ -147,7 +147,7 @@ export const useContentTaxonomyTagsUpdater = (contentId) => {
         contentPattern = contentId.replace(/\+type@.*$/, '*');
       }
       queryClient.invalidateQueries({ queryKey: ['contentTagsCount', contentPattern] });
-      if (contentId.includes('lb:')) {
+      if (contentId.includes('lb:') || contentId.includes('lib-collection:')) {
```

Comment:
nit (for consistency):
```suggestion
      if (contentId.startsWith('lb:') || contentId.startsWith('lib-collection:')) {
```

Reply:
Updated: https://github.com/openedx/frontend-app-authoring/pull/1379/commits/534c5ab41b069443cb931d378f176106f38ceebc